_inc/connect-button.js

```diff
@@ -110,7 +109,7 @@ jQuery( document ).ready( function ( $ ) {
 			document.head.appendChild( link );
 		},
 		fetchPlanType: function () {
-			$.ajax( {
+			return $.ajax( {
```

Comment:
There's an effort to get rid of jQuery in the repo, so while you're at it, it might make sense to replace `$.ajax` with `XMLHttpRequest`.
Completely up to you though.